TombEngine/Objects/TR3/Entity/tr3_tribesman.cpp

```diff
@@ -385,6 +386,7 @@ namespace TEN::Entities::Creatures::TR3
 		short tilt = 0;
 		auto extraHeadRot = EulerAngles::Zero;
 		auto extraTorsoRot = EulerAngles::Zero;
+		short torso = 0;
```

Comment:
What is the purpose of this variable?

Reply:
oh, this was a remnant of one of my tests. I removed it.